bundles/org.openhab.binding.mqtt.homeassistant/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="mqtt" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Is it expected to use id mqrt and not an id about homeassistant?

Reply:
You're correct, the id is wrong. This would override the main org.openhab.binding.mqtt